src/components/Attachment/AttachmentContainer.tsx

```diff
@@ -72,11 +81,41 @@ export const GalleryContainer = <
 >({
   attachment,
   Gallery = DefaultGallery,
-}: RenderGalleryProps<StreamChatGenerics>) => (
-  <AttachmentWithinContainer attachment={attachment} componentType='gallery'>
-    <Gallery images={attachment.images || []} key='gallery' />
-  </AttachmentWithinContainer>
-);
+}: RenderGalleryProps<StreamChatGenerics>) => {
+  const imageElements = useRef<HTMLElement[]>([]);
+  const { imageAttachmentSizeHandler } = useChannelStateContext();
+  const [attachmentConfigurations, setAttachmentConfigurations] = useState<
```

Comment:
Is it necessary to keep the whole configuration? Wouldn't it suffice to keep the `previewUrl`?